manifests/params.pp

```diff
@@ -2,6 +2,7 @@
 class certs::params {
 
   $pki_dir = '/etc/pki/katello'
+  $nss_db_dir   = "${pki_dir}/nssdb"
```

Comment:
This means it's no longer possible to change `$pki_dir` for `$nss_db_dir`, which it previously was. That's a regression.